ScanbotSDKExamples/NET/ClassicComponent.iOS/ViewControllers/VINScannerViewController.cs

```diff
@@ -0,0 +1,48 @@
+// This file has been autogenerated from a class added in the UI designer.
+
+using ScanbotSDK.iOS;
+
+namespace ClassicComponent.iOS
+{
+    interface IVINScannerDelegate
+    {
+        void ShowResult(string result);
+    }
+
+    public partial class VINScannerViewController : UIViewController, IVINScannerDelegate
+    {
+        public VINScannerViewController(IntPtr handle) : base(handle)
+        {
+        }
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+            var configuration = SBSDKVehicleIdentificationNumberScannerConfiguration.DefaultConfiguration();
+            _ = new SBSDKVINScannerViewController(parentViewController: this,
```

Comment:
@scan-mayank, I think we should store this controller instance in a local variable as storing it in the function scope may lead to it going out of scope unexpectedly.